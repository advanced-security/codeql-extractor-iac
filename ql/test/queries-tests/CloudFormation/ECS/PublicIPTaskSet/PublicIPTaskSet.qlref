security/CloudFormation/ECS/PublicIPTaskSet.ql
