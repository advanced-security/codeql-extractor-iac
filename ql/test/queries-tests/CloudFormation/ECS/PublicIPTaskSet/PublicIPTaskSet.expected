| ecs.yml:5:5:37:2 | CloudFormation ECS Task Set | AssignPublicIp must be "DISABLED" for ECS tasks |
